packages/cadl-ranch-specs/http/resiliency/dev-driven/main.cadl

```diff
@@ -78,9 +80,8 @@ This method requires to write 2 tests.
   - With DPG 2.0, generate the convenience method to pass Input("world!") and read Product model with "received" to "model"
 """)
 @scenario
-@route("/customization/model/{mode}")
+@route("/model/{mode}")
```

Comment:
Couldn't we just duplicate the operation, have it exactly the same but say pass raw data in one and pass a model in the other